src/client/pages/Embark/addressAutocompleteProvider.ts

```diff
@@ -17,13 +29,17 @@ const query = gql`
   }
 `
 
-export const resolveAddressAutocomplete = async (
+export const resolveAddressAutocomplete: AddressAutocompleteQuery = async (
   term: string,
-): Promise<AddressSuggestion[]> => {
+  { type } = { type: 'STREET' },
+) => {
```

Comment:
Do we want to specify return type or is that unnecessary? :)

Reply:
It's already defined in the type `AddressAutocompleteQuery`